Ruleset [r2c-ci] fails to validate

**Describe the bug**
The `r2c-ci` ruleset fails to validate due to a colon in the message for `python.lang.security.audit.network.bind.avoid-bind-to-all-interfaces`. It appears to be fixed in the [repo](https://github.com/returntocorp/semgrep-rules/blob/83a5edffa6c547e77935bed0dba1941f7cc8117f/python/lang/security/audit/network/bind.yaml#L4) (if I'm looking in the right place), but not deployed.

Same for `r2c-security-audit`.

**To Reproduce**
Steps to reproduce the behavior:
1. Download the ruleset: `wget -O r2c-ci.yml https://semgrep.dev/c/p/r2c-ci`
2. Validate it: `semgrep -f r2c-ci.yml --validate`

Ouptut:
```
Configuration is invalid - found 1 configuration error(s), and 0 rule(s).
[ERROR] Invalid YAML file r2c-ci.yml:
	while parsing a block mapping
	  in "<file>", line 1559, column 3
	expected <block end>, but found '<scalar>'
	  in "<file>", line 1560, column 47
```

Offending snippet:
![image](https://github.com/returntocorp/semgrep-rules/assets/732166/183cdfc6-d00d-4847-85f9-71bd4d517191)

**Expected behavior**
The ruleset should validate. If I remove the colons from line 1560, then it does so:

```
Configuration is valid - found 0 configuration error(s), and 92 rule(s).
If Semgrep missed a finding, please send us feedback to let us know!
  $ semgrep shouldafound --help
```

**Priority**
How important is this to you?
- [ ] P0: blocking me from making progress
- [ ] P1: this will block me in the near future
- [X] P2: annoying but not blocking me

**Screenshots**
See above.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Linux
 - Browser [e.g. chrome, safari] N/A
 - Version [e.g. 22]: Semgrep 1.24.1

**Additional context**
I also noticed that `--validate` doesn't seem to work with `--config` at all -- I tried multiple rulesets.

![image](https://github.com/returntocorp/semgrep-rules/assets/732166/4cc05f3d-954f-4a74-8273-3bbcb0595367)

--- a/python/lang/security/audit/network/bind.yaml
+++ b/python/lang/security/audit/network/bind.yaml
@@ -1,7 +1,7 @@
 rules:
 - id: avoid-bind-to-all-interfaces
   message: >-
-    Running `socket.bind` to 0.0.0.0, ::, or empty string could unexpectedly
+    Running `socket.bind` to 0.0.0.0, or empty string could unexpectedly
     expose the server publicly as it binds to all available interfaces. Consider
     instead getting correct address from an environment variable or
     configuration file.

